downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,353 @@
+- Added the `persist migrate` command to generate migration scripts for data model changes in the `mysql` data store. The support for migrations is currently an experimental feature and its behavior may be subject to change in future releases.
+
+#### Language Server
+
+- Added inlay hint support for function call expressions and method call expressions to provide information about parameters.
+
+### Improvements
+
+#### Test Framework
+
+- Added support for excluding source files from code coverage.
+
+#### Persist Tool
+
+- Added support for additional DB configurations in the generated client objects for the `mysql` data store.
+- Changed the Ballerina `byte[]` type mapping to `LONGBLOB` in the generated SQL script file.
+
+#### Language Server
+
+- Removed service template initialization from the lightweight mode.
+- Improved the completion support and signature help for client resource access actions.
+- Improved the main function completion item.
+- Improved completions in the named argument context.
+- Added support to rename parameter documentation for record fields and required parameters.
+
+#### Ballerina Update Tool
+
+- Improved the `bal dist update` command to update the active distribution to the latest patch version (of the active Swan Lake version). 
+- Improved the `bal dist pull latest` command to update the active distribution to the latest Swan Lake version.
+
+#### OpenAPI Tool
+
+- Added support for regular expression templates (i.e., the `pattern` property) defined on OpenAPI `string` types in the Ballerina client and service generation.
+  With this support, the aforementioned Regex templates will be represented as `ballerina/constraint` module annotations in the generated code. This support applies to Regex patterns that satisfy [the Ballerina regular expression grammar](https://ballerina.io/spec/lang/2022R4/#section_10.1).
+- Added support for OpenAPI enums in the client and service generation. 
+- Added support for query parameters with referenced schema in the Ballerina service generation. 
+- Added support for header parameters with referenced schema in the Ballerina service generation. 
+- Added support for `integer`, `float`, `decimal`, and `boolean` header parameter types in the Ballerina service generation.
+
+#### Architecture model generator
+
+To view improvements of the architecture model generator, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-dev-tools/pulls?q=is%3Apr+is%3Aclosed+label%3AArea%2FArchitectureModelGenerator+label%3AType%2FImprovement+milestone%3A2201.6.0).
+
+#### CLI commands
+
+##### Support for providing paths with `bal new`
+
+Added support to provide a directory path with the `bal new` command to create a package in a specific directory (e.g., `bal new <package-path>`). 
+
+##### Deprecation of `bal init`
```

Comment:
Seems a new L3 sub title `Deprecations` is added under standard library to handle similar info. Why not here?

Reply:
Fixed in the below.

- https://github.com/ballerina-platform/ballerina-dev-website/pull/6975#discussion_r1209037423
- https://github.com/ballerina-platform/ballerina-dev-website/pull/6975#discussion_r1209037684

Reply:
Fixed.